packages/ckeditor5-ui/src/textarea/textareaview.ts

```diff
@@ -22,24 +22,136 @@ export default class TextareaView extends InputBase<HTMLTextAreaElement> {
 	 * @observable
 	 * @default 1
 	 */
-	declare public rows: number;
+	declare public minRows: number;
+
+	/**
+	 * Specifies the maximum number of rows.
+	 *
+	 * @observable
+	 * @default 5
+	 */
+	declare public maxRows: number;
+
+	/**
+	 * TODO
+	 *
+	 * @observable
+	 * @default null
+	 */
+	declare public height: number | null;
+
+	/**
+	 * TODO
+	 *
+	 * @observable
+	 * @default 'both'
+	 */
+	declare public resize: 'both' | 'horizontal' | 'vertical' | 'none';
 
 	/**
 	 * @inheritDoc
 	 */
 	constructor( locale?: Locale ) {
 		super( locale );
 
-		this.set( 'rows', 1 );
+		const toPx = toUnit( 'px' );
+
+		this.set( 'minRows', 1 );
+		this.set( 'maxRows', 5 );
```

Comment:
I implemented a validation logic, this makes sense. 

Unfortunately, we cannot just pass these as static values to the constructor because we're using (passing) the `createLabeledTextarea` helper down the `LabeledFieldView` and when it's ready, it would be too late for anyone to customize anything. With observables, we can change these at any time, also when the `AutocompleteView` is up and ready.